Lesson_15/geometric_homework.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        self.setattr('side_a', side_a)
+        self.setattr('angle_a', angle_a)
+
+    def setattr(self, name, value):
+        if name == 'side_a':
+            if value <= 0:
+                raise ValueError("The side of the rhombus must be greater than 0")
+            self.__dict__[name] = value
+
+        elif name == 'angle_a':
```

Comment:
Also instead of using all this logic inside 1 method to do it less complicated - please create 2 separate private methods for setting attributes. Only call them inside your main __setattr__ method. Thanks.